migrations/20200201140355_initial.up.sql

```diff
@@ -2,6 +2,9 @@ CREATE TABLE uris (
   uri text PRIMARY KEY
 );
 
+/* here and further "ON <operation> RESTRICT" options make foreign key checks non deferrable,
+ * so that one cannot change data even if application stack is hypothetically compromised
```

Comment:
s/if application stack is hypothetically compromised/if server database credentials are compromised/